packages/editor/src/components/collab-sidebar/utils.js

```diff
@@ -7,3 +7,21 @@
 export function sanitizeCommentString( str ) {
 	return str.trim();
 }
+
+/**
+ * Gets the background color of the editor canvas.
+ *
+ * @return { string } Background color of the editor canvas.
+ */
+export function getEditorCanvasBackgroundColor() {
+	const iframe = document.querySelector( 'iframe[name="editor-canvas"]' );
+	const iframeDocument =
+		iframe?.contentDocument || iframe?.contentWindow.document;
+	const iframeBody = iframeDocument?.body;
+	if ( iframeBody ) {
+		const style = window.getComputedStyle( iframeBody );
+		return style?.backgroundColor;
```

Comment:
It would be nice if we can just get the colour from global styles, but I guess this is fine for now.

Reply:
I’ve updated the implementation to pull the color directly from the global styles., and also kept this function for fallback background color.

Reply:
Let's remove it for now, and reduce complexity. There's also cases where the editor doesn't have an iframe, so this code is too fragile atm.

Reply:
Got it, I'll remove the fallback function for now. However, we might still encounter cases where global styles aren't supported, should we consider that later?